### PR TITLE
Update mde-device-control-device-installation.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the note in the correct section

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10880

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/mde-device-control-device-installation.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ If you disable or don't configure this policy setting, Windows can install and u
...
 
 > [!NOTE]
-> To enable the **Allow installation of devices using drivers that match these device setup classes**, **Allow installation of devices that match any of these device IDs**, and **Allow installation of devices that match any of these device instance IDs** policy settings to supersede this policy setting for applicable devices, enable the **Apply layered order of evaluation for Allow and Prevent device installation policies across all device match criteria** policy setting.
+> To enable the **Allow installation of devices using drivers that match these device setup classes**, **Allow installation of devices that match any of these device IDs**, and **Allow installation of devices that match any of these device instance IDs** policy settings to supersede this policy setting for applicable devices, enable the **Apply layered order of evaluation for Allow and Prevent device installation policies across all device match criteria** policy setting. Also, the allow policy won’t take precedence if the Block Removable Storage option is selected in Device Control.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mde-device-control-device-installation.md]:
```suggestion
> To enable the **Allow installation of devices using drivers that match these device setup classes**, **Allow installation of devices that match any of these device IDs**, and **Allow installation of devices that match any of these device instance IDs** policy settings to supersede this policy setting for applicable devices, enable the **Apply layered order of evaluation for Allow and Prevent device installation policies across all device match criteria** policy setting. Also, the allow policy won’t take precedence if the **Block Removable Storage** option is selected in Device Control.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Review by v-britweston — APPROVED** [review]

**Pull request merged**